### PR TITLE
Ref [Task #44444] admin auto-logout

### PR DESCRIPTION
**Pull request by hieunn-2956**

### CODE REVIEW

**Comment by hieunn-2956** [comment]
ready

File: admin/src/App.js [hunk 1]
```diff
@@ -7,22 +7,29 @@ import Order from "./pages/Order";
...
 
   useEffect(() => {
     if (!isAuthenticate) {
       dispatch(isUserLoggedIn());
     }
   }, [isAuthenticate]);
 
+  useEffect(() => {
+    if (expiresIn < new Date().getTime()) {
+      console.log("expires");
```
Comment (manhnt-0960) [inline admin/src/App.js]: bỏ log

File: admin/src/sagas/authSagas.js [hunk 2]
```diff
@@ -14,12 +14,17 @@ const logIn = async (user) => {
...
+    token: response.data.token,
+    user: response.data.user,
+    expiresIn: response.data.expiresIn.then,
+  };
 };
 
 export function* logInWithCredentials({ payload: { email, password } }) {
   try {
     const user = yield logIn({ email, password });
+    console.log(user);
```
Comment (manhnt-0960) [inline admin/src/sagas/authSagas.js]: bỏ log

**Comment by hieunn-2956** [comment]
ready

**Pull request merged**